icaruscode/Decode/DecoderTools/TriggerDecoderV3_tool.cc

```diff
@@ -811,7 +803,31 @@ namespace daq
     return { s.data(), std::min(s.find_first_of(endl), s.size()) };
   }
   
-
+  
+  sbn::ExtraTriggerInfo::CryostatInfo TriggerDecoderV3::unpackPrimitiveBits(
+    std::size_t cryostat, bool firstEvent, unsigned long int counts,
+    std::uint64_t connectors01, std::uint64_t connectors23
+  ) {
+    sbn::ExtraTriggerInfo::CryostatInfo cryoInfo;
+    
+    // there is (or was?) a bug on the first event in the run,
```

Comment:
Not directly for this code change, but Is this still the case? If so we might want to understand where that is introduced.

Reply:
Where it is introduced: it is part of the original trigger packet, so it must come from the FPGA code. Massimo Rossella did not identify the issue, but I don't think he devoted much time to that.
I don't know whether the issue is still present; in the few logs I have around I see no issue except in a 2022 run, and even at that time not all the runs showed that issue (many did not).